style.css

```diff
@@ -150,13 +150,13 @@ p {
 }
 
 .lang-container {
-  display: flex;
   list-style-type: none;
   padding: 0;
```

Comment:
Why weren’t you interested in giving value to padding? This is because sometimes it can cause a problem when you upload it to Github.